### PR TITLE
adding housekeeping scripts

### PR DESCRIPTION
**Pull request by bam241**
Updating news check to use GithubAction

switching to unified  CHANGELOG entries

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,7 +2,39 @@
 cyclus Change Log
 =================
 
-.. current developments
+Since last release
+====================
+
+**Added:**
+
+* adding compatibility with cbc 2.10
```
Comment (gonuke) [inline CHANGELOG.rst:10]: Should we put PR numbers on each entry?

**Review by gonuke — CHANGES_REQUESTED** [review]
This is supposed to fail the news test on CircleCI, right?  Should we remove that test as part of this PR?

File: CHANGELOG.rst (hunk 1 above)
Comment (bam241) [reply CHANGELOG.rst:10]: we did not in the past.
I can do it for the coming release, and set it as a expectation...

Comment (bam241) [reply CHANGELOG.rst]: I actually thought it is better to have the PR numbers, so I added them.

**Comment by bam241** [comment]
I also removed housekeeping from CircleCI

**Review by gonuke — APPROVED** [review]
Thanks @bam241

**Pull request merged**